app/controllers/schedules_controller.rb

```diff
@@ -56,7 +58,7 @@ def schedule_params
     params.require(:schedule)
           .permit(:SCHEDULE_API_KEY, :schedule_run_id,
                   run_data: [:case_num, :hmc_score,
-                             assignments: [:week, :proposal]])
+                             { assignments: %i[week proposal] }])
```

Comment:
I know that this is RuboCop's preferred ruby syntax -- but is it valid Strong Parameters syntax?